docs/app/documentation/component-docs/input/input-docs.component.html

```diff
@@ -0,0 +1,55 @@
+<h2>Input</h2>
+<component-example [name]="'ex1'">
+    <fd-input-example></fd-input-example>
+</component-example>
+<code-example [exampleFiles]="inputsFormHtml"></code-example>
+
+<separator></separator>
+
+<h2>Input help elements</h2>
+<component-example [name]="'ex2'">
+    <fd-input-inline-help-example></fd-input-inline-help-example>
+</component-example>
+<code-example [exampleFiles]="inputsHelpFormHtml"></code-example>
+
+<separator></separator>
+
+<h2>Input States</h2>
+<description>
+    The state of the input field can reflect validity of the data entered, whether the input data is editable or
+    disabled.
+    <br />
+    <b>Normal:</b> The field is editable but no validation has occurred.
+    <br />
+    <b>Valid:</b> The data format entered has been validated and it’s correct, such as an email address.
+    <br />
+    <b>Invalid:</b> The data entered is not valid and must be corrected.
+    <br />
+    <b>Warning:</b> The data entered is formatted correctly but there are other issues are problematic but will not stop
```

Comment:
The description is not very clear.

Reply:
I'm just gonna delete these descriptions, it's leftover after old forms example.